README.md

```diff
@@ -138,10 +415,46 @@ In order to run the account-service in development mode without containers, you
 
 **Note:** Reference `.vscode/launch.json` for more details on the debug configurations and apply the concepts to your preferred debugger.
 
-## Architecture
 
-The account-service is a NestJS application that is split into two main parts: the API and the Worker.
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
 
-The API is responsible for handling incoming HTTP requests and the Worker is responsible for processing jobs that require blockchain interaction.
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+- [Contributing Guidelines](./CONTRIBUTING.md)
+- [Open Issues](https://github.com/AmplicaLabs/acccount-service/issues)
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support
+
+If you like this project...loom at the label for `good-first-issue`
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓FAQ
+
+- **[Question_1]**
+
+    - [Answer_1]
+
+- **[Question_2]**
+
+    - [Answer_2]
```

Comment:
Do we want to add anything to the FAQ? How this connects to social app template? Thoughts?

Reply:
I would keep FAQs to the `account-service` and not mention SAT. But here are some possible FAQs we could answer:

Q: Can I use this service in my production social app?
A: Yes. All of the Gateway Services are intended to be ready-to-use out of the box as part of the fabric of your own social media app using DSNP on Frequency. [licensing specifics?]

Q: I'm building the next Facebook, it's going to be huge! Will `account-service` scale?
A: Gateway Services are designed to support the scale of a small-to-medium-sized social app [any stats on expected size--number of users, tx/sec, etc?]. For larger use cases, you would probably want to build your own services.

Reply:
@wilwade any notes on metrics?